src/favoriteRestaurant.js

```diff
@@ -1,7 +1,7 @@
 // Write functions here
```

Comment:
To find the largest value, you don't need to sort the array (which is a O(n lg n) operation) you just need to iterate over the array using a linear search. (which is a O(n) operation)

Finding the second largest value can also be done in the same loop.